bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1202 @@
+## @section Global parameters
```

Comment:
These values were not in the template (NOTE THAT THERE ARE FALSE POSITIVES, make sure you review)

```
Value global.imageRegistry not found in template
Value global.imagePullSecrets not found in template
Value kubeVersion not found in template
Value clusterDomain not found in template
Value service.port not found in template
Value service.httpsPort not found in template
Value service.nodePorts not found in template
Value service.nodePorts.http not found in template
Value service.nodePorts.https not found in template
Value service.clusterIP not found in template
Value service.annotations not found in template
Value concourse.image.debug not found in template
Value web.bindIp not found in template
Value web.encryption not found in template
Value web.encryption.enabled not found in template
Value web.readinessProbe.httpGet not found in template
Value web.readinessProbe.httpGet.path not found in template
Value web.readinessProbe.httpGet.port not found in template
Value web.tls.bindPort not found in template
Value web.existingConfigmap not found in template
Value web.podSecurityContext.fsGroup not found in template
Value web.containerSecurityContext.runAsUser not found in template
Value web.service.api.clusterIP not found in template
Value web.service.api.loadBalancerIP not found in template
Value web.service.api.labels not found in template
Value web.service.api.annotations not found in template
Value web.service.api.loadBalancerSourceRanges not found in template
Value web.service.api.tlsNodePort not found in template
Value web.service.workerGateway.labels not found in template
Value web.service.workerGateway.annotations not found in template
Value web.tsa.clientConfig not found in template
Value web.tsa.clientConfig.enabled not found in template
Value web.tsa.hostKey not found in template
Value web.tsa.atcUrl not found in template
Value web.tsa.heartbeatInterval not found in template
Value web.tsa.gardenRequestTimeout not found in template
Value web.auth.cookieSecure not found in template
Value web.auth.duration not found in template
Value web.auth.passwordConnector not found in template
Value web.auth.mainTeam.config not found in template
Value web.auth.mainTeam.userIDFieldPerConnector not found in template
Value web.ingress.hosts not found in template
Value web.ingress.rulesOverride not found in template
Value worker.healthcheckBindIp not found in template
Value worker.healthcheckBindPort not found in template
Value worker.livenessProbe.failureThreshold not found in template
Value worker.livenessProbe.initialDelaySeconds not found in template
Value worker.livenessProbe.periodSeconds not found in template
Value worker.livenessProbe.timeoutSeconds not found in template
Value worker.livenessProbe.httpGet not found in template
Value worker.livenessProbe.httpGet.path not found in template
Value worker.livenessProbe.httpGet.port not found in template
Value worker.podSecurityContext.fsGroup not found in template
Value worker.containerSecurityContext.privileged not found in template
Value worker.nodeAffinityPreset.affinity not found in template
Value worker.nodeAffinityPreset.nodeSelector not found in template
Value worker.nodeAffinityPreset.tolerations not found in template
Value worker.baggageclaim.reapInterval not found in template
Value worker.baggageclaim.p2pInterfaceFamily not found in template
Value worker.baggageclaim.p2pInterfaceNamePattern not found in template
Value volumePermissions.containerSecurityContext not found in template
Value volumePermissions.containerSecurityContext.runAsUser not found in template
Value secrets.create not found in template
Value secrets.annotations not found in template
Value secrets.webTlsCert not found in template
Value secrets.webTlsKey not found in template
Value secrets.webTlsCaCert not found in template
Value persistence.worker.annotations not found in template
Value persistence.worker.accessModes not found in template
Value externalDatabase.user not found in template
Value externalDatabase.password not found in template
Value externalDatabase.database not found in template
```